src/Engine/ProtoCore/CodeGen.cs

```diff
@@ -1516,11 +1521,11 @@ protected void EmitBounceIntrinsic(int blockId, int entry)
             instr.opCode = ProtoCore.DSASM.OpCode.CALLR;
             instr.op1 = StackValue.BuildFunctionIndex(fi); 
             instr.op2 = StackValue.BuildClassIndex(type);
-            instr.op3 = StackValue.BuildInt(depth);
```

Comment:
It is used. Previously the 3rd operand of `callr` is depth, and block id is pushed to the stack, now the 3rd operand of `callr` is block id, and depth is pushed to the stack.